daily-challenges/2024-09/14/tunganh222Z_1409_BMI.java

```diff
@@ -0,0 +1,32 @@
+import java.util.Scanner;
+
+public class tunganh222Z_1409_BMI {
+
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Vui lòng nhập số cân nặng kg của bạn");
+        double weight = sc.nextDouble();
+        System.out.println("Vui lòng nhập số chiều cao m của bạn");
+        double height = sc.nextDouble();
+
+
+        System.out.println("Chỉ số BMI của bạn là :" + caculateBMI(weight, height));
+
+    }
+
+        public static String caculateBMI ( double inputWeight, double inputHeight){
+
+            double BMI = (inputWeight / Math.pow(inputHeight, inputHeight));
```

Comment:
```suggestion
            double BMI = inputWeight / (inputHeight * inputHeight);
```

Về cơ bản chỗ này không sai. Nhưng với phép tính đơn giản thì mình nghĩ Tùng Anh nên viết ở dạng simple là phép nhân thôi cho dễ hiểu.

Ngoài ra thì ngoặc tròn bao ngoài cùng bị thừa nha.